Script/AtomicEditor/ui/frames/inspector/InspectorFrame.ts

```diff
@@ -213,6 +214,22 @@ class InspectorFrame extends ScriptWidget {
             prefabInspector.inspect(asset);
         }
 
+        if (asset.importerTypeName == "TextureImporter") {
```

Comment:
This file looks like it could be a good candidate for refactoring out the inspectors into some kind of service locator.  Probably not necessary yet, but as more inspectors get defined, it could be useful.